zz.home-work/src/App.test.tsx

```diff
@@ -1,3 +1,46 @@
-describe('Todo List', () => {
-  test('-', () => {});
+import { describe, test } from "vitest"
+import { render, fireEvent } from "@testing-library/react";
+import App from "./App.tsx";
+
+describe('Todo Input의 글자 수는 100자 미만이어야 한다', () => {
+  const isTodoValid = (text: string) => text.length < 100;
+
+  test('99자 일 때 테스트 해보기', () => {
+    expect(isTodoValid("1".repeat(99))).toEqual(true)
```

Comment:
p5;
`toBeTruthy` 요것도 조금 명확할 것 같네요 :)
```suggestion
    expect(isTodoValid("1".repeat(99))).toBeTruthy(true)
```